### PR TITLE
admin create instructor page: Add some explanations about the forms #4357

### LINKED ISSUE #4357: admin create instructor page: Add some explanations about the forms
Can add some explanations about why there are two forms, the format used for first form, etc.
This extra info will be useful for devs who are accessing TEAMMATES dev/staging server as admin.

image

### PR DESCRIPTION
**Pull request by Pulasthih**
Fixes #4357

Added headings to the two forms explaining why each form is used and added an explanation on how to add multiple users to the first form.

### CODE REVIEW

**Comment by bamboo3250** [comment]
@Pulasthih Could you upload a screenshot?

File: src/main/webapp/WEB-INF/tags/admin/home/adminCreateInstructorAccountWithOneBoxForm.tag [hunk 1]
```diff
@@ -4,7 +4,11 @@
 
 <div class="well well-plain">
     <div>
-        <label class="label-control">Instructor Details: (Name | Email | Institution)</label>
+        <label class="label-control">Adding Multiple Instructors</label>
+        <div class="text-muted">
+            <span class="glyphicon glyphicon-exclamation-sign glyphicon-primary"></span> Add Instructor Details in the format (Name | Email | Institution)
+        </div>
+        <br>
```
Comment (bamboo3250) [inline src/main/webapp/WEB-INF/tags/admin/home/adminCreateInstructorAccountWithOneBoxForm.tag]: Change to `<br/>` to make it consistent?

Comment (wkurniawan07) [inline src/main/webapp/WEB-INF/tags/admin/home/adminCreateInstructorAccountWithOneBoxForm.tag]: No, our coding standard specifies `<br>`. It's the other one that needs edit.

**Comment by Pulasthih** [comment]
Screenshot of the change is given below. 
b522b204-e3dd-11e5-99a5-ed21b36c05ff

File: src/main/webapp/WEB-INF/tags/admin/home/adminCreateInstructorAccountForm.tag [hunk 2]
```diff
@@ -7,6 +7,9 @@
 
 <div class="well well-plain">
     <div>
+        <label class="label-control">Adding a Single Instructor</label>
+    </div><br/>
```
Comment (bamboo3250) [inline src/main/webapp/WEB-INF/tags/admin/home/adminCreateInstructorAccountForm.tag]: Sorry, my mistake. All `<br/>` should be `<br>` according to our HTML coding style guide. https://docs.google.com/document/d/12PJYbQoqjK-0LzaUuguQ4kGE--eikCcHfwzZDGwFOJ0/pub?embedded=true

Comment (Pulasthih) [inline src/main/webapp/WEB-INF/tags/admin/home/adminCreateInstructorAccountForm.tag]: Noted. Will do the changes. Is a line break required before the <br> or can it be inline with the </div>

Comment (bamboo3250) [inline src/main/webapp/WEB-INF/tags/admin/home/adminCreateInstructorAccountForm.tag]: Leaving it inline may not break the rule but I think putting it to a new line looks better. Let's do the same for below.

**Comment by bamboo3250** [comment]
Now you need to run `AdminHomePageUiTest` and see if it fails due to the change in HTML or not (very likely). If that happens, please use Godmode to fix them. https://github.com/TEAMMATES/teammates/blob/master/devdocs/godmode.md

After that, run all tests and see if there is any other test that also fails due to this change or not then fix it like we did for AdminHomePageUiTest.

**Comment by thyageshm** [comment]
@bamboo3250 I am afraid this is a FirstTimers issue. Could you help him with fixing the tests?

**Comment by Pulasthih** [comment]
Ran the `AdminHomePageUiTest` and it failed as expected due to the change in HTML giving a comparison failure. Then used Godmode to fix it. Now everything works fine.

**Comment by thyageshm** [comment]
@bamboo3250 for your review

**Comment by bamboo3250** [comment]
@Pulasthih I couldn't see your commit. Could you commit and push it?

**Comment by thyageshm** [comment]
@Pulasthih please push your commit (again?) as the changes to the tests are not visible in this PR

**Comment by thyageshm** [comment]
@bamboo3250 ready to review

**Pull request merged**